### ISSUE TITLE
Time::parse causes a strange output

### ISSUE THREAD

**Issue by phoniclynx** [duncan3dc/sonos#85]
Passing the time in HH:mm to Time:parse with the string "16:19" gives the alarm an incorrect time value of 16 mins past midnight ("00:16") using the line
`$alarm->setTime(Time::parse($time));`

To correct this I have to add the seconds into the line like so:
`$alarm->setTime(Time::parse($time.":00"));`

Should Time:parse not be detecting that it is in HH:mm and not HH:mm:ss and correct itself purhaps?

**Comment by duncan3dc** [comment]
Hmm, it's difficult as `parse()` has no way of knowing with `16:19` is 4pm, or 16 minutes and 19 seconds.

I wonder if a `TIme::fromFormat()` would be useful, something like:
```php
$time = Time::fromFormat("h:m", "16:19");

$time = Time::fromFormat("h:m:s", "16:19:00");
```

**Comment by phoniclynx** [comment]
yeah that could work

**Comment by duncan3dc** [comment]
Sorry for the delay, this is now available in version [2.0.3](https://github.com/duncan3dc/sonos/releases/tag/2.0.3)
